src/cli.ts

```
@@ -1135,6 +1140,233 @@ function openInBrowser(filePath: string): void {
+    );
+  }
+  if (summary.majorUpgradeBlockerBreakdown.installScripts > 0) {
+    blockerDetails.push(
+      `   - ${summary.majorUpgradeBlockerBreakdown.installScripts} ${pluralize(
+        summary.majorUpgradeBlockerBreakdown.installScripts,
+        "install lifecycle script",
+        "install lifecycle scripts",
+      )}`,
+    );
+  }
+  blockerDetails.forEach((line) => console.log(line));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)